src/main/java/com/example/quily/services/KGSService.java

```diff
@@ -36,4 +38,12 @@ public synchronized Mono<KGSResponse> getCurrentKey() {
         return Mono.error(
                 new ResourceNotFoundException("unable to fetch keyIndices of id: " + id + " from DB"));
     }
+
+    public Mono<KeyIndices> saveCurrentKey(KGSRequest kgsRequest) {
+
+      KeyIndices updatedIndices =  new KeyIndices( (long)kgsRequest.getId(),kgsRequest.getIndex1(),kgsRequest.getIndex2(),kgsRequest.getIndex3(),kgsRequest.getIndex4(),kgsRequest.getIndex5(),kgsRequest.getIndex6());
```

Comment:
parameter should separate with , and extra space and switch to next line if content is too long in single line.
```suggestion
      KeyIndices updatedIndices =  new KeyIndices( (long)kgsRequest.getId(), kgsRequest.getIndex1(), kgsRequest.getIndex2(), kgsRequest.getIndex3(), kgsRequest.getIndex4(), kgsRequest.getIndex5(), kgsRequest.getIndex6());
```